packages/pip.go

```diff
@@ -34,7 +34,7 @@ var (
 
 func init() {
 	if runtime.GOOS != "windows" {
-		pip = "/usr/bin/pip"
+		pip = "/usr/bin/pip3"
```

Comment:
This isn't safe as for all distros that we support, also we are not sure if pip is something we want to continue to scan by default because of a number of issues we have with it.